pkg/services/ngalert/notifier/alertmanager.go

```diff
@@ -1,3 +1,4 @@
+//nolint:golint,unused
```

Comment:
Let's remember to drop before merge

Reply:
We can't drop it - In order to keep the diff minimal, there are now a lot of attributes and functions that remain unused. My next PR shortly after this one will take care of removing this linter skip and delete all the dead code.